### DISCUSSION TITLE
call app inside a mutation (WebSocket)

### DISCUSSION THREAD

**Discussion by DrakoPOD** [quasarframework/quasar#8856] [General - Components / Directives / etc]
I'm trying to do a WebSocket connection, but the following line I don't know how to implement
`main.config.globalProperties.$socket = event.currentTarget;`
This line is inside of a mutation, I think that instead of `main` is `app`.

I'm following this example: [vue-native-websocket-vue3](https://github.com/likaia/vue-native-websocket-vue3/blob/master/README-EN.md)

**Comment by metalsadman** [comment]
Do that in a boot file https://next.quasar.dev/quasar-cli/boot-files#introduction

**Reply by DrakoPOD** [reply]
Is not that easy, because I also need to change a state.
